src/main/java/net/minestom/server/instance/lighting/ChunkLight.java

```diff
@@ -0,0 +1,155 @@
+package net.minestom.server.instance.lighting;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.instance.Chunk;
+import net.minestom.server.network.packet.server.play.UpdateLightPacket;
+
+/**
+ * Configure chunk light data.
+ * <p>
+ * Notably used in {@link ChunkLightEngine#lightChunk(ChunkLight)} when implementing {@link ChunkLightEngine}
+ */
+public class ChunkLight {
```

Comment:
This class shouldn't exist in my opinion. It should just be a property of every Chunk and should only live there. 

In a more ideal world, this would be per chunk section, but there is some consideration of how this will be handled in current and future versions (1.17)

Reply:
So far, in 1.17, the only major change according to wiki.vg is just the addition of array length-- but I don't know much more than that. The idea of making the class specific to sections isn't a bad idea either, I'll make the adjustment.. however, I do believe we should keep it as a wrapper for readability sake and for ease of use.

Reply:
There is some question of how much Minestom will change to use sections over chunks, and this will depend somewhat on that.

In terms of keeping it as a separate class, I am curious of other's opinions, but I feel it fits with the rest of the chunk style choices for it to be a property of Chunk. 

cc @TheMode